pkg/channel/consolidated/status/status.go

```diff
@@ -467,6 +509,15 @@ func (m *Prober) probeVerifier(item *workItem) prober.Verifier {
 	}
 }
 
+// A target state is outdated if the generation is different or if the target IPs change before it becomes
+// ready.
+func isOutdatedTargetState(s *targetState, sub eventingduckv1.SubscriberSpec, podIPs sets.String) bool {
+	s.readyLock.RLock()
+	defer s.readyLock.RUnlock()
+	r := s.ready
```

Comment:
is adding a single use, single char variable like this for just the next line really more readable than just using `s.ready` ?

Reply:
nope, that function changed a lot and that one was overlooked